parser/filter_test.go

```diff
@@ -19,9 +21,9 @@ func TestFilterConnPairWithInternalSubnets(t *testing.T) {
 		res:             nil,
 		indexingThreads: 1,
 		parseThreads:    1,
-		internal:        getParsedSubnets([]string{"10.0.0.0/8"}),
-		alwaysIncluded:  getParsedSubnets([]string{"10.0.0.1/32", "10.0.0.3/32", "1.1.1.1/32", "1.1.1.3/32"}),
-		neverIncluded:   getParsedSubnets([]string{"10.0.0.2/32", "10.0.0.3/32", "1.1.1.2/32", "1.1.1.3/32"}),
+		internal:        util.ParseSubnets([]string{"10.0.0.0/8"}),
+		alwaysIncluded:  util.ParseSubnets([]string{"10.0.0.1/32", "10.0.0.3/32", "1.1.1.1/32", "1.1.1.3/32"}),
```

Comment:
I don't know what the usage of these are but just a note that there are subnets that appears in both alwaysIncluded and neverIncluded (10.0.0.3/32, 1.1.1.3/32)

Reply:
These are intentional for tests that check which list takes precedent.